src/grm/main.rs

```diff
@@ -199,7 +199,7 @@ fn main() {
                         }
                     };
 
-                    let (found_repos, warnings) = match find_in_tree(&path) {
+                    let (found_repos, warnings) = match find_in_tree(&path, &args.exclude) {
```

Comment:
When changing `find_repos()` to take `Option<&str>`, you'll also have to update this. Now the question is, how do we transform `Option<String>` to `Option<&str>`?

Luckily, `String` implements `Deref<Target = str>`, so we can use a method already defined on `Option`:

```suggestion
                    let (found_repos, warnings) = match find_in_tree(&path, args.exclude.as_deref()) {
```